### DISCUSSION TITLE
Welcome to Green-Coder Discussions!

### DISCUSSION THREAD

**Discussion by GreenCityLife** [GreenCityLife/Green-Coder#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inari-codes** [comment]
omg, green coder??? 😳 cool

**Comment by GreenCityLife** [comment]
Yes 😳

**Comment by inari-codes** [comment]
but uh... why you said that in README..
>  My only goal is to make advanced Code editor **with outdated sketchware**
>
🤔
"with original sketchware" is better for me imo

**Reply by GreenCityLife** [reply]
Oh yeah thanks

**Reply by Abhinandan080** [reply]
Lol

**Comment by Abhinandan080** [comment]
Cool, but Errors

**Reply by GreenCityLife** [reply]
It would be more cool to post the errors you faced in issues
